android/src/main/java/com/dylanvann/fastimage/FastImageViewManager.java

```diff
@@ -124,7 +130,17 @@ public void setSrc(FastImageViewWithUrl view, @Nullable ReadableMap source) {
                     .load(imageSource.getSourceForLoad())
                     .apply(FastImageViewConverter.getOptions(context, imageSource, source))
                     .listener(new FastImageRequestListener(key))
-                    .into(view);
+                    .into(new DrawableImageViewTarget(view) {
+                        @Override
+                        public void onResourceReady(@NonNull Drawable resource, @Nullable Transition<? super Drawable> transition) {
+                            super.onResourceReady(resource, transition);
+                            if (resource instanceof WebpDrawable) {
+                                WebpDrawable webpDrawable = (WebpDrawable) resource;
+                                webpDrawable.setLoopCount(1);
+                                webpDrawable.stop();
+                            }
+                        }
+                    });
```

Comment:
Instead of stopping the animation directly after it's started, we should probably just load the first frame of the animation as suggested [here](https://github.com/bumptech/glide/issues/1107#issuecomment-205735933) .

Reply:
Do we have to do this because we don't want the animation to start immediately but on demand?